src/routes/docs/contributing/translating/+page.md

```diff
@@ -0,0 +1,106 @@
+---
+
+Translations for Files are done through the popular cloud based platform Crowdin. Anyone with a Crowdin account can translate strings and help improve existing strings.
+
+## Getting started with Crowdin
+
+- Any browser that work on Windows with the latest version
+- [Crowdin account](https://accounts.crowdin.com/register)
+
+Once you translate a string, proofreaders or maintainers rarely requests changes, but if you notice an issue with an approved translation, you can use the Comments section in Crowdin Editor with the Issue check box marked.
+
+## Proofread
+
+If you translate Files on a regular basis and would like to have access to proofread your language, please start a discussion on Crowdin and we'll review your request at the earliest convenience.
+
+## Screenshots
+
+We add some screenshots where possible to help translators understand better under what context strings are used.
+When you're translating a string in Crowdin Editor, Crowdin automatically shows screenshots and highlightes target.
+
+## See in Files
+
+Maintainers often merge all translations that are approved right before a preview release or stable release published. Once gets merged and gets new release published, you can see translated words on your Windows.
+
+## Placeholders
+
+Numbers that are surrounded by curly brackets are the placeholder where actual values are set at runtime. Translators may have to consider under what context string are used, and so we sometimes add comments about at which sentences are basically shown and what they are for as long as possible.
+
+For example:
+
+```
+Are you sure you want to delete {0} from {1}?
+```
+
+In this case, the first placeholder is for file name that user wants to delete and second one is for folder name that user wants to delete a file from.
+
+## ICU message syntax
+
+You may see some strings that have programmatic syntax.
+It’s called ICU message syntax, which is the standard syntax advanced by Unicode (and the parallel ISO 10646 standard) for efficiently processing text in any language and for maintaining text data integrity
+
+### Pluralization
+
+Plural type is used to handle plural category variations, as each language has its own set of plural categories (for example, English has “one” and “other”, while Ukrainian has “one”, “few”, “many”, and “other”).
+
+The following short tags are used to determine the plural categories:
+
+- zero
+- one (singular)
+- two (dual)
+- few (paucal)
+- many (also used for fractions if they have a separate class)
+- other (required—general plural form—also used if the language only has a single form)
+
+In the Editor, you don’t have to manually add or delete plural categories to the translations you are making. Just click `Copy Source`, and the source string will be copied to the translation field with the number of plural categories right for the current target language.
+
+![image](https://github.com/0x5bfa/Website/assets/62196528/830ef3f0-945b-4dae-8381-00805c7bd590)
```

Comment:
Can you include this in the website repo?

Reply:
Sure! Are they should be light themed?

Reply:
Either way is fine

Reply:
Done